### PR TITLE
Explain the predicates as suffixes convention

### PR DESCRIPTION
**Pull request by YaelDillies**
This convention was widely applied but not written down apart from the special case of `Injective`.

### CODE REVIEW

File: templates/contribute/naming.md [hunk 1]
```diff
@@ -416,3 +415,11 @@ automatically generated unidirectional implications, named `.inj`,
...
 When such an automatically generated lemma already exists,
 and a bidirectional lemma is needed, it may be named `.inj_iff`.
+
+### Predicates as suffixes
+
+Most predicates should be added as prefixes. Eg `IsClosed (Icc a b)` should be called `isClosed_Icc`, not `Icc_isClosed`.
+
+Some widely used predicates escape this rule. Those are the predicates that are analogous to an atom already suffixed by the naming convention. Here is a non-exhaustive list:
+* We use `_inj` for `f a = f b ↔ a = b`, so we also use `_injective` for `Injective f`, `_surjective` for `Surjective f`, `_bijective` for `Bijective f`...
+* We use `_mono` for `a ≤ b → f a ≤ f b` and `_anti` for `a ≤ b → f a ≤ f b`, so we also use `_mono` for `Monotone f``, `_anti` for `Antitone f`, `_strictMono` for `StrictMono f`, `_strictAnti` for `StrictAnti f`, etc...
```
Comment (urkud) [inline templates/contribute/naming.md:425]:
```suggestion
* We use `_mono` for `a ≤ b → f a ≤ f b` and `_anti` for `a ≤ b → f b ≤ f a`, so we also use `_mono` for `Monotone f``, `_anti` for `Antitone f`, `_strictMono` for `StrictMono f`, `_strictAnti` for `StrictAnti f`, etc...
```

**Review by urkud — APPROVED** [review]

File: templates/contribute/naming.md [hunk 2]
```diff
@@ -416,3 +415,11 @@ automatically generated unidirectional implications, named `.inj`,
 and there is no intention to change this.
 When such an automatically generated lemma already exists,
 and a bidirectional lemma is needed, it may be named `.inj_iff`.
+
+### Predicates as suffixes
+
+Most predicates should be added as prefixes. Eg `IsClosed (Icc a b)` should be called `isClosed_Icc`, not `Icc_isClosed`.
+
+Some widely used predicates escape this rule. Those are the predicates that are analogous to an atom already suffixed by the naming convention. Here is a non-exhaustive list:
```
Comment (kbuzzard) [inline templates/contribute/naming.md:423]:
```suggestion
Some widely used predicates don't follow this rule. Those are the predicates that are analogous to an atom already suffixed by the naming convention. Here is a non-exhaustive list:
```
I would love that this were correct English but unfortunately I don't think that this lovely French idiom can be literally translated.

Comment (YaelDillies) [reply templates/contribute/naming.md:423]: You are chasing the little beast!

File: templates/contribute/naming.md [hunk 3]
```diff
@@ -416,3 +415,11 @@ automatically generated unidirectional implications, named `.inj`,
...
 When such an automatically generated lemma already exists,
 and a bidirectional lemma is needed, it may be named `.inj_iff`.
+
+### Predicates as suffixes
+
+Most predicates should be added as prefixes. Eg `IsClosed (Icc a b)` should be called `isClosed_Icc`, not `Icc_isClosed`.
+
+Some widely used predicates don't follow this rule. Those are the predicates that are analogous to an atom already suffixed by the naming convention. Here is a non-exhaustive list:
+* We use `_inj` for `f a = f b ↔ a = b`, so we also use `_injective` for `Injective f`, `_surjective` for `Surjective f`, `_bijective` for `Bijective f`...
+* We use `_mono` for `a ≤ b → f a ≤ f b` and `_anti` for `a ≤ b → f b ≤ f a`, so we also use `_mono` for `Monotone f``, `_anti` for `Antitone f`, `_strictMono` for `StrictMono f`, `_strictAnti` for `StrictAnti f`, etc...
```
Comment (fpvandoorn) [inline templates/contribute/naming.md:425]:
```suggestion
* We use `_mono` for `a ≤ b → f a ≤ f b` and `_anti` for `a ≤ b → f b ≤ f a`, so we also use `_mono` for `Monotone f`, `_anti` for `Antitone f`, `_strictMono` for `StrictMono f`, `_strictAnti` for `StrictAnti f`, etc...
```

**Pull request merged**